docs/mkdocs/docs/developers/paig-client/code-examples/milvus-integration/index.md

```diff
@@ -0,0 +1,77 @@
+---
```

Comment:
we can consider putting this under more generic Vector Search section as it is applicable for all vector databases
later along with milvus, we can put opensearch example also

Reply:
<img width="297" alt="image" src="https://github.com/user-attachments/assets/dc83dbdc-5e5c-406c-aabc-0a8731692660">
Updated the navigation as below